### PR TITLE
Tutorial page redesign

### PR DESCRIPTION
**Pull request by colleenmcginnis**
This replaces the current [Tutorials page layout for the Maps SDK](https://www.mapbox.com/android-docs/map-sdk/tutorials). This uses a new `TutorialCard` component modeled after the design of the [Unity documentation's tutorial page](https://www.mapbox.com/unity-sdk/tutorials/) so we can link to multiple step-by-step tutorials hosted on the [Help page](https://www.mapbox.com/help/). @jfurrow let me know if there's a better way to go about building a layout like this!

Still to do: 
- [x] decide on section titles (i.e. `Getting started`, `Annotations` in the screenshot below) @cammace @langsmith 
- [x] review by @jfurrow 
- [x] review by @cammace 
- [x] remove unused images @colleenmcginnis

Note: The DDS tutorial isn't live yet so the link will likely 404. (edit: it's live now 🎉 )

image

### CODE REVIEW

File: src/components/tutorial-card.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '@mapbox/react-icon';
+import { AppropriateImage } from './appropriate-image';
+import { DifficultyLevel } from './difficulty-level';
+
+class TutorialCard extends React.Component {
+  static propTypes = {
+    tutorialUrl: PropTypes.string.isRequired,
+    tutorialImgID: PropTypes.string.isRequired
```
Comment (jfurrow) [inline src/components/tutorial-card.js]: You could define the rest of the `props` that are used here, too, if you'd like. `propTypes` are validated during development, so defining all of the `props` could be helpful to future developers. I think the missing props are `difficulty`, `tutorialTitle`, and `tutorialDescription`.

**Comment by jfurrow** [comment]
@colleenmcginnis This looks great to me!

**Comment by langsmith** [comment]
@colleenmcginnis , the current section titles are 👍 to me

**Comment by langsmith** [comment]
@colleenmcginnis , what do you think about my creating a run-time styling `tut` as well? Could be nice to have before the data-driven styling one...

**Pull request merged**